.github/workflows/auto-labeling.yml

```
@@ -0,0 +1,201 @@
+            const labelsToAdd = new Set();
+
+            // PRの場合のみファイル変更ベースでラベリング
+            if (context.eventName === 'pull_request' && changedFiles) {
+              const files = changedFiles.split(' ');
+              
+              for (const [labelName, rule] of Object.entries(labelRules)) {
+                const shouldAddLabel = rule.patterns.some(pattern => {
+                  if (pattern.startsWith('!')) {
+                    // 除外パターン
+                    const excludePattern = pattern.slice(1);
+                    return !files.some(file => 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)